server/protocols/webadmin/webadmin-data/src/test/java/org/apache/james/webadmin/routes/GroupsRoutesTest.java

```diff
@@ -319,6 +324,26 @@ void putShouldDetectLoops() {
                 .containsEntry("message", "Creation of redirection of group1@b.com to group:a@b.com would lead to a loop, operation not performed");
         }
 
+        @Test
+        void putShouldNotConflictWithAlias() throws Exception {
+            memoryRecipientRewriteTable.addAliasMapping(MappingSource.fromUser(Username.of(GROUP1)), USER_A);
+
+            Map<String, Object> errors = when()
+                .put(GROUP1 + SEPARATOR + USER_A)
+            .then()
+                .contentType(ContentType.JSON)
+                .statusCode(HttpStatus.CONFLICT_409)
+                .extract()
+                .body()
+                .jsonPath()
+                .getMap(".");
+
+            assertThat(errors)
+                .containsEntry("statusCode", HttpStatus.CONFLICT_409)
+                .containsEntry("type", "WrongState")
+                .containsEntry("message", "'group1@b.com' already have associated mappings: alias:a@b.com");
+        }
+
```

Comment:
Where are tests for:

```
GIVEN a group marketing@linagora.com
WHEN I create the user marketing@linagora.com
THEN it fails
```

```
GIVEN a group marketing@linagora.com
WHEN I create the group marketing@linagora.com
THEN it fails
```
?

Reply:
```
GIVEN a group marketing@linagora.com
WHEN I create the group marketing@linagora.com
THEN it fails
```

Actually no: you only add member to a group. And of course a group can have several members.

```
GIVEN a group marketing@linagora.com
WHEN I create the group marketing@linagora.com
THEN it fails
```
This is covered in user route tests...